Core/include/Acts/TrackFitting/GainMatrixUpdater.hpp

```diff
@@ -61,39 +49,42 @@ class GainMatrixUpdater {
 
     // read-only handles. Types are eigen maps to backing storage
     const auto predicted = trackState.predicted();
-    const auto predicted_covariance = trackState.predictedCovariance();
+    const auto predictedCovariance = trackState.predictedCovariance();
 
     ACTS_VERBOSE("Predicted parameters: " << predicted.transpose());
-    ACTS_VERBOSE("Predicted covariance:\n" << predicted_covariance);
+    ACTS_VERBOSE("Predicted covariance:\n" << predictedCovariance);
 
     // read-write handles. Types are eigen maps into backing storage.
     // This writes directly into the trajectory storage
     auto filtered = trackState.filtered();
-    auto filtered_covariance = trackState.filteredCovariance();
+    auto filteredCovariance = trackState.filteredCovariance();
 
-    std::optional<std::error_code> error{std::nullopt};  // assume ok
+    // default error represents an invalid non-error state, no need for optional
```

Comment:
This comment makes no sense out of context of the changeset.

Reply:
Fixed.